src/components/MapView/Layers/BoundaryDropdown.tsx

```diff
@@ -369,9 +369,10 @@ export const GotoBoundaryDropdown = () => {
     | undefined;
   const { data } = boundaryLayerData || {};
 
+  const navigationDropdown = get(appConfig, 'navigationDropdown', false);
```

Comment:
another option would be to extend the type of appConfig

Reply:
or move this logic to `prism-frontend/src/config/index.ts` wdyt?

Reply:
I moved it to the config index file here: fa5e2eb57e893887d3dec610d1b192fe1b797247